web/massastation/src/pages/Index/Index.tsx

```diff
@@ -39,18 +39,17 @@ export function Index() {
   const { data: availablePlugins } =
     useResource<PluginStoreItemRequest[]>('plugin-store');
 
-  const { mutate, isSuccess, isError } = usePost<any, any>(
-    `plugin-manager?source=${urlPlugin}`,
-  );
+  const { mutate, isSuccess, isError } = usePost<any, any>(`plugin-manager`);
```

Comment:
can you specify the type ?

Reply:
good catch!

Reply:
I don't know the type. The request doesn't return anything except a status.